contrib/src/main/java/com/twelvemonkeys/contrib/tiff/TIFFUtilities.java

```diff
@@ -510,13 +660,11 @@ public void rotate(int degree) {
             }
 
             newIDFData.add(new TIFFEntry(TIFF.TAG_ORIENTATION, (short) orientation));
-            IFD = new AbstractDirectory(newIDFData) {};
+            IFD = new AbstractDirectory(newIDFData) {
```

Comment:
Sorry, missed this the first time, but you can now use the public `com.twelvemonkeys.imageio.metadata.tiff.IFD` class, instead of the anonymous class.